src/Storage/Storage/Table/Cmdlet/GetStorageAzureTable.cs

```diff
@@ -53,6 +60,11 @@ public class GetAzureStorageTableCommand : StorageCloudTableCmdletBase
         [ValidateNotNullOrEmpty]
         public string Prefix { get; set; }
 
+        [Parameter(HelpMessage = "Table Query",
+            ParameterSetName = QueryParameterSet, Mandatory = true)]
+        [ValidateNotNullOrEmpty]
+        public string Query { get; set; }
```

Comment:
Please also update the help doc for the cmdlet

Reply:
remove query parameter and will add back when we deprecate track 1 sdk